dlp/snippets/deid.py

```diff
@@ -1282,6 +1415,45 @@ def deidentify_with_exception_list(
         help="The list of strings to ignore matches against.",
     )
 
+    table_condition_mask_parser = subparsers.add_parser(
+        "deid_table_condition_mask",
+        help="De-identify sensitive data in a table by masking"
+        "them based on a condition.",
+    )
+    table_condition_mask_parser.add_argument(
+        "project",
+        help="The Google Cloud project id to use as a parent resource.",
+    )
+    table_condition_mask_parser.add_argument(
+        "table_data",
+        help="Json string representing table data",
+    )
+    table_condition_mask_parser.add_argument(
+        "deid_content_list",
+        help="A list of fields in table to de-identify."
+    )
+    table_condition_mask_parser.add_argument(
+        "--condition_field",
+        help="A table Field within the record this condition is evaluated "
+        "against.",
+    )
+    table_condition_mask_parser.add_argument(
+        "--condition_operator",
```

Comment:
Some of the arguments are present in another PR as well. Needs resolving conflicts once previous PR is merged.

Reply:
Arguments are for different functions, there is no duplication. and hence there won't be any conflict